Modules/HR/Observers/Recruitment/JobObserver.php

```diff
@@ -21,16 +21,17 @@ public function created(Job $job)
         if (! config('database.connections.wordpress.enabled')) {
             return;
         }
-
         $job->rounds()->attach(Round::pluck('id')->toArray());
         $data = request()->all();
         $corcel = new Corcel();
         $corcel->post_title = $data['title'];
         $corcel->post_content = $data['description'];
         $corcel->post_type = config('hr.post-type.career');
         $corcel->post_name = str_replace(' ', '-', strtolower($data['title']));
-        if ($data['status'] != 'published') {
-            $corcel->post_status = 'draft';
+        foreach (config('hr.opportunities-status-mapping') as $status => $label) {
+            if ($data['status'] == $status) {
+                $corcel->post_status = $label;
+            }
         }
```

Comment:
I think this can be replaced by:
```suggestion
        $corcel->post_status = config('hr.opportunities-status-mapping')[$data['status']]
```

Do you agree?